src/components/Button/PlayButton.js

```diff
@@ -51,7 +54,7 @@ const PlayButton = ({ hover=false, type, expanded, className }) => {
 PlayButton.propTypes = {
   hover: PropTypes.bool,
   type: PropTypes.string,
-  expanded: PropTypes.string
+  expanded: PropTypes.oneOf([false, 'horizontal', 'vertical'])
```

Comment:
I'm leaving myself a note to git rid of the boolean there at some point and check for a falsey value instead